src/App.js

```diff
@@ -1,16 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatEntry from './components/ChatEntry';
+import ChatLog from './components/ChatLog';
+
 
 const App = () => {
+
+  const [chatData, setChatData] = useState(chatMessages);
+  let [likeCount, setLikeCount] = useState(0);
+  // let likeCount = 0;
```

Comment:
Gentle reminder we want to clean up commented out code so it doesn't get into our final codebase.